.github/workflows/github-actions.yml

```
@@ -84,3 +84,8 @@ jobs:
     uses: ./.github/workflows/project-workflow.yml
     with:
       project: baseapp-chats
+  baseapp:
+    uses: ./.github/workflows/project-workflow.yml
+    with:
+      project: baseapp
+      custom_tests_path: "baseapp"
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)